cmd/iam.go

```diff
@@ -591,11 +592,25 @@ func (sys *IAMSys) SetTempUser(accessKey string, cred auth.Credentials, policyNa
 	// temporary user which match with pre-configured canned
 	// policies for this server.
 	if globalPolicyOPA == nil && policyName != "" {
-		p, ok := sys.iamPolicyDocsMap[policyName]
-		if !ok {
-			return errInvalidArgument
+		var availablePolicies []iampolicy.Policy
+		for _, pname := range strings.Split(policyName, ",") {
+			if pname == "" {
+				continue
+			}
+			p, found := sys.iamPolicyDocsMap[pname]
```

Comment:
```suggestion
			pname = strings.TrimSpace(pname)
			if pname == "" {
				continue
			}
			p, found := sys.iamPolicyDocsMap[pname]
```

We can trim whitespace to make the format more forgiving.

Reply:
Sure this can be done @donatello